### PR TITLE
Update CHANGES_NEXT_RELEASE (java version)

### PR DESCRIPTION
**Pull request by AlvaroVega**
comes from https://github.com/telefonicaid/fiware-cygnus/pull/2284

### CODE REVIEW

File: CHANGES_NEXT_RELEASE [hunk 1]
```diff
@@ -3,4 +3,5 @@
 - [cygnus-common][SQL] Fix expiration records select with a limit to avoid java out of memory error (#2273)
 - [cygnus-ngsi] Removes "_" in schema name for DM -schema family (#2270, #2201 reopens for Postgis)
 - [cygnus-ngsi] UPGRADE: Debian version from 11.6 to 12.1 in Dockerfile
+- [cygnus-ngsi] UPGRADE: Java 11 to Java 17 used in Dockerfile
```
Comment (fgalan) [inline CHANGES_NEXT_RELEASE:6]: Java 17 should be enabled in ci.yml (currently, the line is commented)

Comment (AlvaroVega) [reply CHANGES_NEXT_RELEASE:6]: Java 16 and Java 17 tests are failing due to some tests has problems with class instantation of :
mockito 
java.regex

Comment (AlvaroVega) [reply CHANGES_NEXT_RELEASE:6]: https://github.com/telefonicaid/fiware-cygnus/issues/2297

**Comment by AlvaroVega** [comment]
https://github.com/telefonicaid/fiware-cygnus/pull/2305

**Pull request closed without merging**